api/app/fire_behaviour/advisory.py

```diff
@@ -0,0 +1,173 @@
+                 bui: float, ffmc: float, isi: float, wind_speed: float, wind_direction: float,
+                 temperature: float, relative_humidity: float, precipitation: float, status: str,
+                 prev_day_daily_ffmc: float, last_observed_morning_rh_values: dict):
+        self.elevation = elevation
+        self.fuel_type = fuel_type
+        self.time_of_interest = time_of_interest
+        self.percentage_conifer = percentage_conifer
+        self.percentage_dead_balsam_fir = percentage_dead_balsam_fir
+        self.grass_cure = grass_cure
+        self.crown_base_height = crown_base_height
+        # not many people know/care about the crown fuel load, so we can fill this out with default
+        # values if set to None.
+        self.crown_fuel_load = crown_fuel_load
+        self.lat = lat
+        self.long = long
+        self.bui = bui
+        self.ffmc = ffmc
+        self.isi = isi
+        self.wind_speed = wind_speed
+        self.wind_direction = wind_direction
+        self.temperature = temperature
+        self.relative_humidity = relative_humidity
+        self.precipitation = precipitation
+        self.status = status
+        self.prev_day_daily_ffmc = prev_day_daily_ffmc
+        self.last_observed_morning_rh_values = last_observed_morning_rh_values
+
+    def __str__(self) -> str:
+        return f"lat {self.lat}, long {self.long}, elevation {self.elevation}, fuel_type {self.fuel_type}, \
+            time_of_interest {self.time_of_interest}, percentage_conifer {self.percentage_conifer},\
+            percentage_dead_balsam_fir {self.percentage_dead_balsam_fir}, grass_cure {self.grass_cure},\
+            crown_base_height {self.crown_base_height}, crown_fuel_load {self.crown_fuel_load}, bui {self.bui},\
+            ffmc {self.ffmc}, isi {self.isi}, prev_day_daily_ffmc {self.prev_day_daily_ffmc}, wind_speed {self.wind_speed},\
+            temperature {self.temperature}, relative_humidity {self.relative_humidity}, \
+            precipitation {self.precipitation}, status {self.status}"
+
+
+class FireBehaviourAdvisory():  # pylint: disable=too-many-instance-attributes
+    """ Class containing the results of the fire behaviour advisory calculation. """
+
+    def __init__(self,  # pylint: disable=too-many-arguments
+                 hfi: float, ros: float, fire_type: FireTypeEnum, cfb: float, flame_length: float,
+                 sixty_minute_fire_size: float, thirty_minute_fire_size: float,
+                 critical_hours_hfi_4000: Optional[CriticalHoursHFI],
+                 critical_hours_hfi_10000: Optional[CriticalHoursHFI],
+                 hfi_t: Optional[float],
+                 ros_t: Optional[float],
+                 cfb_t: Optional[float],
+                 sixty_minute_fire_size_t: Optional[float]):
+        self.hfi = hfi
+        self.hfi_t = None
+        self.ros = ros
+        self.ros_t = None
+        self.fire_type = fire_type
+        self.cfb = cfb
+        self.cfb_t = None
```

Comment:
Why are the `_t` values being assigned None here, when a few lines below they're being assigned the value passed to init?